### PR TITLE
feat: added category component

### PR DESCRIPTION
**Pull request by eakansh125**

### CODE REVIEW

File: screens/category.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    border-radius: 10px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    flex-wrap: wrap;
+}
+
+.category-card{
+    width: 250px;
+    height: 420px;
```
Comment (Sahil25061999) [inline screens/category.css]: use rem

File: screens/category.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    border: 1px solid var(--secondary-grey);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.category-img{
+    width: 100%;
+    height: 220px;
```
Comment (Sahil25061999) [inline screens/category.css]: here as well.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            </div>
+            <div class="nav-header-div-mid">
+                <input class="input-outlined" type="text" placeholder="Search" />
+            </div>
+            <div class="nav-header-div-rt">
+
+                <a href="#" class="btn-small margin-lr">Login</a>
+                <img class="pointer" src="https://img.icons8.com/material/36/000000/user-male-circle--v1.png" />
+
+            </div>
```
Comment (Ankur9669) [inline index.html:26]: fix: 
1. Add alt tag to the image
2. Remove extra spacing

File: index.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+            <div class="home-container">
+                <div class="quiz-home-lt">
+                    <h1>Welcome to the Fleet play quiz, how about you just keep that phone away for a while and have some fun?</h1>
+                    <h4>Click the button below to dicover the categories of quiz</h4>
+                    <a href="screens/category.html" class="btn-small margin-lr">Take the quiz</a>
+                </div>
+                <div class="quiz-home-rt">
+                    <img class="quiz-image" src="img/undraw_quiz_nlyh.svg" alt="quiz-image">
+                </div>
```
Comment (Ankur9669) [inline index.html]: fix: add alt tag

**Review by Sahil25061999 — APPROVED** [review]
Looks good ready to merge with minor changes.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+            </div>
+
+
+        </div>
+
```
Comment (Ankur9669) [inline index.html]: fix: remove extra spacing

File: screens/category.css [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    object-fit: cover;
```
Comment (Ankur9669) [inline screens/category.css:27]: Suggestion: If possible use rem here, by the way this is also fine, it is your choice

File: screens/category.html [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+            </div>
+            <div class="nav-header-div-mid">
+                <input class="input-outlined" type="text" placeholder="Search" />
+            </div>
+            <div class="nav-header-div-rt">
+
+                <a href="#" class="btn-small margin-lr">Login</a>
+                <img class="pointer" src="https://img.icons8.com/material/36/000000/user-male-circle--v1.png" />
+
+            </div>
```
Comment (Ankur9669) [inline screens/category.html:27]: fix:
1. Add alt tag
2. Remove extra spacing

File: screens/category.html [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+                    </div>
+
+                </div>
+
+                
+            </div>
+
+
+        </div>
+
```
Comment (Ankur9669) [inline screens/category.html]: fix: There is a lot of extra spacing that can be removed

File: screens/rules.html [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+            </div>
+            <div class="nav-header-div-mid">
+                <input class="input-outlined" type="text" placeholder="Search" />
+            </div>
+            <div class="nav-header-div-rt">
+
+                <a href="#" class="btn-small margin-lr">Login</a>
+                <img class="pointer" src="https://img.icons8.com/material/36/000000/user-male-circle--v1.png" />
+
+            </div>
```
Comment (Ankur9669) [inline screens/rules.html]: Same

**Review by Ankur9669 — APPROVED** [review]
UI Suggestion:
image

The borders are very light and are very hard to see please make them dark


The code is good but there is a lot of extra spacing and alt tags are missing

File: screens/category.css (hunk 1 above)
Comment (eakansh125) [reply screens/category.css]: Done

**Pull request merged**